app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt

```diff
@@ -148,24 +141,7 @@ fun EventInfoSheet(
                 modifier = Modifier.align(Alignment.BottomCenter).padding(bottom = 20.dp),
             ) {
                 // button to join the event
-                Button(
-                    enabled = isOnline,
-                    onClick = {
-                        onJoinButtonPressed()
-                        Toast.makeText(
-                                context,
-                                context.getString(R.string.event_successfully_joined),
-                                Toast.LENGTH_SHORT
-                            )
-                            .show()
-                    },
-                    modifier = Modifier.testTag("join_button_event_info_sheet"),
-                ) {
-                    Text(
-                        text = stringResource(R.string.event_info_sheet_join_event_button_text),
-                        style = MaterialTheme.typography.labelLarge
-                    )
-                }
+                JoinEventButton(event, isOnline, 100.dp)
```

Comment:
This should be something like 130.dp, otherwise the button text needs more than 1 line...